data-pipeline/src/trace_exporter.rs

```diff
@@ -310,23 +457,110 @@ impl TraceExporterBuilder {
         self
     }
 
-    pub fn build(mut self) -> anyhow::Result<TraceExporter> {
+    /// Set the header indicating the tracer has computed the top-level tag
+    pub fn set_client_computed_top_level(mut self) -> Self {
+        self.client_computed_top_level = true;
+        self
+    }
+
+    /// Set the header indicating the tracer has already computed stats.
+    /// This should not be used when stats computation is enabled.
+    pub fn set_client_computed_stats(mut self) -> Self {
+        self.client_computed_stats = true;
+        self
+    }
+
+    /// Enable stats computation on traces sent through this exporter
+    pub fn enable_stats(mut self, bucket_size: time::Duration) -> Self {
+        self.stats_bucket_size = Some(bucket_size);
```

Comment:
Ah so if we have `Some` bucket_size that's how we know stats is enabled?

Reply:
Yes that's it. I can add a comment to make it explicit